shoes-swt/lib/shoes/swt/button.rb

```diff
@@ -11,6 +11,11 @@ def initialize(dsl, app)
           button.set_text @dsl.text
         end
       end
+
+      def text=(value)
+        @last_text = @real.text
```

Comment:
None at all. I did it based on edit lines, and preferred to be conservative by keeping that line, but I've been sweeping through the code and I think it can be removed. I'll fix that in a PR including specs.